apps/src/blockly/addons/cdoFieldImageDropdown.ts

```diff
@@ -191,6 +191,8 @@ export class CdoFieldImageDropdown extends FieldGridDropdown {
       );
       if (this.menuGenerator_.length === 1) {
         this.EDITABLE = false;
+        // No-op showing the dropdown editor since there is only one option.
+        this.showEditor_ = () => {};
```

Comment:
We no-op here to avoid showing this pointless dropdown when there's only one option:

<img width="343" height="150" alt="image" src="https://github.com/user-attachments/assets/9a4a42bd-a0ac-4785-b610-bf1ff448bef3" />